### PR TITLE
Added support for Strip-Utopija

### PR DESCRIPTION
**Pull request by ZerocksX**
https://github.com/tachiyomiorg/tachiyomi-extensions/issues/3384

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [ ] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [ ] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: src/all/striputopija/build.gradle [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlinx-serialization'
+
+ext {
+    extName = 'StripUtopija'
+    pkgNameSuffix = 'all.striputopija'
```
Comment (alessandrojean) [inline src/all/striputopija/build.gradle:7]: Why under the `all` package if the language inside the class is `en`? Move stuff to the `en` folder.

File: src/all/striputopija/src/eu/kanade/tachiyomi/extension/all/striputopija/StripUtopija.kt [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+        val rb = Request.Builder()
+        rb.url(url)
+        rb.header("accept", "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9")
+        rb.header("user-agent", "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36")
+        return rb
+    }
+
+    private fun defaultRequest(url: String): Request {
+        return defaultRequestBuilder(url).build()
+    }
```
Comment (alessandrojean) [inline src/all/striputopija/src/eu/kanade/tachiyomi/extension/all/striputopija/StripUtopija.kt:36]: Why not using the `headersBuilder()`, `headers` and the already provided `GET` and `POST` methods by the extensions' lib?

File: src/all/striputopija/src/eu/kanade/tachiyomi/extension/all/striputopija/StripUtopija.kt [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.Jsoup
+import java.lang.Exception
+import kotlin.collections.ArrayList
+
+class StripUtopija : HttpSource() {
```
Comment (alessandrojean) [inline src/all/striputopija/src/eu/kanade/tachiyomi/extension/all/striputopija/StripUtopija.kt:16]: You're doing a lot of unnecessary code. Would be a lot better to take a look and make the class extends `ParsedHttpSource` instead. More details in the [`CONTRIBUTING.md`](https://github.com/tachiyomiorg/tachiyomi-extensions/blob/master/CONTRIBUTING.md) guide.

**Review by alessandrojean — CHANGES_REQUESTED** [review]
Also use [this tool](https://as280093.github.io/AndroidAssetStudio/icons-launcher.html#foreground.type=clipart&foreground.clipart=android&foreground.space.trim=1&foreground.space.pad=0.25&foreColor=rgba(96%2C%20125%2C%20139%2C%200)&backColor=rgb(68%2C%20138%2C%20255)&crop=0&backgroundShape=square&effects=none&name=ic_launcher) as recommended by the CONTRIBUTING.md guide to generate the icons. They need to follow the same pattern of the other extensions, and not just a logo in a transparent background.

**Comment by arkon** [comment]
@ZerocksX Do you intend on updating this PR?

**Comment by ZerocksX** [comment]
I started working on it a bit. I don't have time for it right now.

**Comment by arkon** [comment]
Closing for now, feel free to reopen if you revisit it.

**Pull request closed without merging**